modules/github/Makefile.init

```diff
@@ -1,6 +1,7 @@
 GITHUB_TEMPLATES = \
 	.github/CODEOWNERS \
 	.github/PULL_REQUEST_TEMPLATE.md \
+	.github/workflows/slash-command-dispatch.yml \
```

Comment:
This is deprecated and replaced by `chatops.yml`

We should make sure that all references to `slash-command-dispatch.yml` are replaced with `chatops.yml`

Reply:
Aha, yeah. This line was a mistake. Good catch.